book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] **A book has many rentals**: A method for adding rentals should be present. 
- Please [Open this link and see examples](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)
- something like this:
```
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```